test/helpers/edge.spec.ts

```diff
@@ -99,12 +93,6 @@ describe('generateRscDataEdgeManifest', () => {
         name: 'RSC data routing',
         pattern: '^/blog/([^/]+?)(?:/)?$',
       },
-      {
-        function: 'rsc-data',
-        generator: '@netlify/next-runtime@1.0.0',
-        name: 'RSC data routing',
-        pattern: '^/blog/([^/]+?)\\.rsc$',
-      },
```

Comment:
Is this removed due to the cache control? Just checking it needs to be deleted for both the feature flag on and off.